sbi/analysis/sensitivity_analysis.py

```diff
@@ -501,3 +502,53 @@ def project(self, theta: Tensor, num_dimensions: int) -> Tensor:
         projected_theta = torch.mm(theta, projection_mat)
 
         return projected_theta
+
+
+def posterior_shrinkage(prior_std, post_std):
```

Comment:
Currently, the function does not do much work.

Maybe just request prior samples and poster samples and do the estimation of the standard deviation within?

Reply:
Type hints should also be here and not in the docstring.